week2/5_String_Methods.ipynb

```diff
@@ -602,11 +622,18 @@
     "</div>"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### <p style=\"text-align: right; direction: rtl; float: right;\">משחקים עם רישיות</p>"
+    "### <p style=\"text-align: right; direction: rtl; float: right;\">משחקים עם רישיות (אותיות גדולות)</p>"
```

Comment:
A bit bloated, I think it is better to remove it.
Maybe change the title to "גדלי אותיות" or something

Reply:
גודלי*